js/index.js

```diff
@@ -1,8 +1,19 @@
-import '../css/style.scss';
+import "../css/style.scss";
```

Comment:
We need to change the default " or ', otherwise it's always going to put them as 'changes' in github. I personally prefer ' in JavaScript